spec/features/site_spec.rb

```diff
@@ -24,5 +24,37 @@
         expect(page).to have_css 'head title', text: 'Custom Name', visible: false
       end
     end
+
+    describe 'institution name' do
+      before do
+        Site.update(
+          institution_name_full: 'fullname'
+        )
+      end
+      it 'updates the institution name in the agreement text' do
+        visit edit_site_path
+        fill_in 'Institution name', with: 'Custom Inst Name'
+        click_on 'Update Site'
+
+        visit sufia.static_path action: 'agreement'
+        expect(page).to have_content('fullname (Custom Inst Name) requires')
```

Comment:
That's what I assumed going in, but the parenthetical seems to be the abbreviation, which also makes sense in its own way. The test also bears that out ("at least on my machine!").